src/system/Wizard/Wizard.js

```diff
@@ -52,6 +70,7 @@ const Wizard = React.forwardRef(
 							subTitle={ subTitle }
 							title={ title }
 							titleVariant={ titleVariant }
+							shouldFocusTitle={ titleAutofocus && activeStep !== initialStep }
```

Comment:
What is the reason for this parameter called `titleAutofocus ` and the other in the other file is `shouldFocusTitle`?

Reply:
Good question! 
`titleAutofocus` is the feature of the component that can be on and off.
`shouldFocusTitle` indicates if/when a single step should focus its title.

Have a better idea for naming? I'm all ears